File: Basics/Prime-or-not/check_prime.py
import math

def check_prime(N):
    for i in range(2, int(math.sqrt(N)) + 1):
        if N % i == 0:
            return False

    return True


n = int(input("Enter the value of N: "))

if n < 2:
    print("Not Prime Number.")
elif check_prime(n):
    print("Prime Number.")
else:
    print("Not Prime Number.")

File: Basics/Prime-or-not/check_prime.cpp
#include<iostream>
using namespace std;

bool check_prime(int N){
    for(int i = 2; i * i <= N; i++){
        if (N % i == 0){
            return false;
        }
    }
    
    return true;
}

int main(){
    int n;
    cout<< "Enter the value of N: ";
    cin>> n;
    if (n < 2){
        cout<< "Not Prime Number.";
    } else if(check_prime(n)){
        cout<< "Prime Number.";
    } else{
        cout<< "Not Prime Number.";
    }

    return 0;
}